src/components/layout/Navbar.tsx

```diff
@@ -42,29 +60,50 @@ const navItems: NavItem[] = [
     label: 'Verification',
     url: '/verification',
   },
+  {
+    label: 'Settings',
+    url: '/settings',
+  },
   {
     label: 'SearchSECO',
     pages: [
-      { label: 'Query', url: '/query' },
-      { label: 'Mining', url: '/mining' },
+      {
+        label: 'Query',
+        url: '/query',
+        icon: HiOutlineDocumentMagnifyingGlass,
+        description: 'Query the SearchSECO database using your $SECOINS',
```

Comment:
Please use the `TOKENS` variable for the symbol of the SECOIN.
Also, we should decide on a consistent way to either always use $ in front of the symbol, or not (currently we do not, but can change if you so desire)